src/cargo/sources/registry/http_remote.rs

```diff
@@ -391,6 +391,25 @@ impl<'cfg> HttpRegistry<'cfg> {
     }
 }
 
+// copied from src/cargo/core/package.rs to keep change minimal
```

Comment:
I think we should avoid copying the macro.  I think it would be fine to just move it somewhere, like `cargo::util::network` or something like that.

Reply:
fixed in https://github.com/rust-lang/cargo/pull/11733/commits/6dbfde37e13c1e899300351d18edd98894d7b627